nhost/compose/ports.go

```diff
@@ -26,3 +63,116 @@ func NewPorts(proxy uint32) Ports {
 	p[SvcTraefik] = proxy
 	return p
 }
+
+// hasura console exposes 2 ports, one for the console itself and one for the migrate api
+func (p Ports) isHasuraConsole(svc string) bool {
```

Comment:
I'd suggest getting rid of this method. You only use it in one place and it is a bit confusing it is associated with the type `Ports` as the comparisons are completely independent with the type itself. If you want to keep it consider making it into a standalone function.